server/controllers/availability.js

```diff
@@ -0,0 +1,67 @@
+const asyncHandler = require('express-async-handler');
+const Availability = require('../models/Availability');
+
+// @route GET /availability/?monday[from]={900}&monday[to]={1300}, etc
+// @desc Get available sitter list based on required customer availability from query
+// @access Private
+exports.getAvailableSitters = asyncHandler(async (req, res, next) => {
```

Comment:
I think this is a good start, we might refactor it to be more clean but good work